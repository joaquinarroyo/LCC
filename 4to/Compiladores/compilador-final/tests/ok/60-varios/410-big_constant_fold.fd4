let x : Nat = 
  let x : Nat = 1 in
  let x : Nat = x + x in
  let x : Nat = x + x in
  let x : Nat = x + x in
  let x : Nat = x + x in
  let x : Nat = x + x in
  let x : Nat = x + x in
  let x : Nat = x + x in
  let x : Nat = x + x in
  let x : Nat = x + x in
  let x : Nat = x + x in
  let x : Nat = x + x in
  let x : Nat = x + x in
  let x : Nat = x + x in
  let x : Nat = x + x in
  let x : Nat = x + x in
  let x : Nat = x + x in
  let x : Nat = x + x in
  let x : Nat = x + x in
  let x : Nat = x + x in
  let x : Nat = x + x in
  let x : Nat = x + x in
  let x : Nat = x + x in
  let x : Nat = x + x in
  let x : Nat = x + x in
  let x : Nat = x + x in
  let x : Nat = x + x in
  let x : Nat = x + x in
  let x : Nat = x + x in
  let x : Nat = x + x in
  let x : Nat = x + x in
  x

let y : Nat = print "x = " x
