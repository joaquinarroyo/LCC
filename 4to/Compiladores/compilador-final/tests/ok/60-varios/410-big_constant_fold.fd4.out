x = 1073741824
